File: C#/SRobotics/Core/RobotCommands.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core
{
    public class RobotCommands
    {
        public RobotCommands()
        {
            TurnOn = new Command.OnCommand();
            SayHello = new Command.HelloCommand();
        }

        public Command.OnCommand TurnOn { get; set; }
        public Command.HelloCommand SayHello { get; set; }
    }
}

File: JAVA/SRobotics/src/sRobotics/core/RobotCommands.java
package sRobotics.core;

import sRobotics.core.command.*;

public class RobotCommands
{
    public RobotCommands()
    {
        turnOn = new OnCommand();
        sayHello = new HelloCommand();
        moveCommand = new MoveCommand();
    }

    public OnCommand turnOn;
    public HelloCommand sayHello;
    public MoveCommand moveCommand;
}
